src/argilla/client/feedback/training/schemas.py

```diff
@@ -381,6 +392,28 @@ def for_chat_completion(
     ) -> "TrainingTaskForChatCompletion":
         return TrainingTaskForChatCompletion(formatting_func=formatting_func)
 
+    @classmethod
+    def for_question_answering(
+        cls,
+        formatting_func: Optional[Callable[[Dict[str, Any]], Union[None, str, List[str], Iterator[str]]]] = None,
```

Comment:
Is this type hint correct? Shouldn't it be tuples?

Reply:
correct